### PR TITLE
Added queueLogMessage to CommandLogger.

### PR DESCRIPTION
**Pull request by noahheller**

### CODE REVIEW

File: src/main/java/org/usfirst/frc4048/common/loggingv2/CommandLogger.java [hunk 1]
```diff
@@ -32,8 +43,27 @@ public void init() {
...
+    private void logQueuedMessages() {
+        long startTime = Logger.getRealTimestamp();
+        Runnable poll = logsToMainThread.poll();
+        while (poll != null){
+            poll.run();
+            long now = Logger.getRealTimestamp();
+            if (now - startTime <= Constants.MAX_LOG_TIME_WAIT){
+                poll = logsToMainThread.poll();
+            } else {
+                Logger.recordOutput("LOGGING_ERROR", "Took To Long @" + now);
```
Comment (armadilloz) [inline src/main/java/org/usfirst/frc4048/common/loggingv2/CommandLogger.java]:
```suggestion
                Logger.recordOutput("LOGGING_ERROR", "Logging took too long @" + now);
```

File: src/main/java/org/usfirst/frc4048/common/loggingv2/CommandLogger.java [hunk 2]
```diff
@@ -46,4 +76,90 @@ public void log() {
             }
         }
     }
+
+    public void queueLogMessage(String key, byte[] value) {
```
Comment (armadilloz) [inline src/main/java/org/usfirst/frc4048/common/loggingv2/CommandLogger.java]:
```suggestion
    public void logMessage(String key, byte[] value) {
```
The fact that this queues something is an implementation detail

**Review by armadilloz — CHANGES_REQUESTED** [review]

File: src/main/java/org/usfirst/frc4048/common/loggingv2/CommandLogger.java (hunk 2 above)
Comment (noahheller) [reply src/main/java/org/usfirst/frc4048/common/loggingv2/CommandLogger.java]: done

File: src/main/java/org/usfirst/frc4048/common/loggingv2/CommandLogger.java (hunk 1 above)
Comment (noahheller) [reply src/main/java/org/usfirst/frc4048/common/loggingv2/CommandLogger.java]: done

**Review by armadilloz — APPROVED** [review]

**Pull request merged**